files/gpg_import_check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+GPG_HOMEDIR=$1
+GPG_KEY_PATH=$2
+
+GPG_PUBKEY=gpg-pubkey-$(echo $(gpg --homedir "${GPG_HOMEDIR}" --with-colons "${GPG_KEY_PATH}" 2>&1 | grep ^pub | awk -F ':' '{print $5}' | cut --characters=9-16 | tr '[:upper:]' '[:lower:]'))
```

Comment:
nit, since you're using `cut` to drop the first 8 characters, maybe use `cut` to print the 5th field instead of `awk`? 

```shell
 ... | grep ^pub | cut -d: -f5 | cut --characters=9-16 | ...
```

Reply:
Yeah, I had to compute the pubkey here since the script didn't like being called with this interpolation. I would have liked to get rid of it in the puppet manifests, but we also need it there.

Reply:
`gpg --homedir "${GPG_HOMEDIR}" --with-colons` could be extracted to a variable for better readibility